### DISCUSSION TITLE
And suddenly, crouton is easy to use once again!!!

### DISCUSSION THREAD

**Discussion by CroutonIsFun** [dnschneid/crouton#5058] [General]
All you have to do is install Chromebrew, then execute this command:

`crew install crew_sudo`

Installing Chromebrew must be done in VT-2, of course. But the above command can be executed in the crosh shell, or VT-2.

Finally, logout of the crosh shell and VT-2. Now log into VT-2 as chronos and the crew_sudo daemon will autostart! Every time, you restart the Chromebook, just log into VT-2 to start the daemon.

And guess what? When you drop to the crosh shell, sudo will work!

YAY!!!

**Comment by DennisLfromGA** [comment]
That's certainly one way.  

I like to use the method outlined below:  
- [chromeos-117-sudo-howto.md](https://gist.github.com/CoolElectronics/a5088c9ade6ec4d35435b9826b45d7a3)  

You have to remove RootFS verification but you don't have to execute it each time you login.  
After a chromeOS update you need to remove RootFS verification & modify the /etc/init/ui.conf file again.

-**DennisLfromGA**

**Reply by CroutonIsFun** [reply]
Thanks, Dennis. Sounds like a more direct solution. Since we crouton fans can so easily find work-arounds to the crosh sudo problem, I'm wondering why the devs took this course of action.

To me, it sounds like a case of "security via obscurity", if you know what I mean. :-)

**Reply by CroutonIsFun** [reply]
From my research, it appears the above would be easier if you could turn on debugging features. This feature is broken right now, but someone included a list of scripts which result from turning on debugging:

[How to manually turn on debugging features](https://support.google.com/chromebook/thread/105387132/i-can-t-turn-on-debugging-features?hl=en)

Apparently, debugging features survive updates, so once you have restored sudo, maybe you will not have to go through the steps again.

**Comment by DennisLfromGA** [comment]
@CroutonIsFun,

> ... but *someone* included a list of scripts which result from turning on debugging

Yup, that 'someone' was me, I'm a chromeOS PE.

-**DennisLfromGA**

**Reply by CroutonIsFun** [reply]
Awesome you!!! Yeah, I went in and noticed your handle, Denny something. You've got a lot of cool stuff in there.

**Reply by CroutonIsFun** [reply]
crew_sudo is now throwing an error in the most recent ChromeOS update: 57, invalid number. The error appears to be mostly cosmetic, and you can still use sudo to execute commands. However, I've quit using it, and I've gone back to ssh into my chroot from the crosh shell.

I'm actually using Crouton to handle situations which Crostini cannot, and I'm using Crostini to run an Emby media server: it just works!

It's great to have alternatives...

**Reply by CroutonIsFun** [reply]
@DennisLfromGA 

AiYiYi --- I'm at version 122.0.6261.137, and the Port Forwarding button is greyed out. I can add a port, but cannot activate it. So I'm back to using Crouton for **everything**.

Edit: After doing some early research in the Crostini SubReddit, I found that users experienced the same issue back in the day after upgrading from Buster to Bullseye. This piqued my curiosity, and lead to discovering a flag which can be set to control the Debian version. Reverting to Bullseye resolves the issue.

**Comment by CroutonIsFun** [comment]
I'm going to rewrite this discussion and provide a more detailed "How-to".
